Shared/mods/deathmatch/logic/luadefs/CLuaCryptDefs.cpp

```diff
@@ -397,6 +407,16 @@ int CLuaCryptDefs::DecodeString(lua_State* luaVM)
                 }
                 return 1;
             }
+            case StringEncryptFunction::BASE32:
+            {
+                lua::Push(luaVM, SharedUtil::Base32decode(data));
+                return 1;
+            }
+            case StringEncryptFunction::BASE64:
+            {
+                lua::Push(luaVM, SharedUtil::Base64decode(data));
+                return 1;
+            }
             default:
             {
                 m_pScriptDebugging->LogCustom(luaVM, "Unknown encryption algorithm");
```

Comment:
Let's change this to "Unknown algorithm" instead.
```suggestion
                m_pScriptDebugging->LogCustom(luaVM, "Unknown algorithm");
```